src/pages/DashboardPage.js

```diff
@@ -8,17 +8,38 @@ import Title from '../compontens/Title';
 import LatestBlogsPage from './LatestBlogsPage';
 import NewBlogPage from './NewBlogPage';
 import MyBlogsPage from './MyBlogsPage';
+import BlogPage from './BlogPage';
+import { useSelector, useDispatch } from 'react-redux';
+import {addBlog, removeBlog} from '../store';
+import {ImProfile} from 'react-icons/im';
+
+export default function DashboardPage({setAuth}) {
+  const dispatch = useDispatch();
+
+  const blogs = useSelector((state)=>{
+    return state.blogs;
+  });
+
+  const handleLogout = (e)=>{
```

Comment:
Unfall xD, why use local storage when you have redux :)

Reply:
I will try to change in next commit.

Reply:
When you said that I can use redux there I was little confused, we can't store token in redux because of fact that state is changing to initial value on every render of our application, we need some way to store our token, and best way is to localStorage instead of db.

Reply:
Token is the data that is going to be constantly sent to backend, you can save it to cookies or local storage, but from my experience it's best if token lives in redux, it's part of the reactive system. Redux is much more flexible than local storage. You can have token in local storage but use it through redux. Save the token in local storage and pull in redux store as initial value so it automatically save with every reload. That was my idea. But if you think that is going to work better go ahead.

Reply:
If we are using redux and local storage, try using [redux-persist](https://www.npmjs.com/package/redux-persist). Why? Well you will skip that part where you manipulate the store and let the package do it for you. Plus you have multiple benefits of redux-persist:

- You can whitelist the Reducer/Splice to have persisted in your app
- You update the state, it instantly puts it in local storage
- Also there is a package that lets you encrypt your persisted store (Which is much more secure than having it plain in local storage )